app/views/discussions/print.html.haml

```diff
@@ -0,0 +1,101 @@
+.print-container
+  .row.title
+    .col-md-9
+      %h1= @discussion.title
+      .groups-title
+        - if @group.is_subgroup?
+          %h3= @group.parent.name
+          = content_tag :span, "\u25B6", class: 'name-separator'
+        %h3=@group.name
+  .row.context
+    .col-md-9
+      = @discussion.description
+    .col-md-2.col-md-offset-1
+      .discussion-details
+        = t("discussion_context.started_print",
+            when: time_ago_in_words(@discussion.created_at),
+            who: @discussion.author.name)
+  - if @motions
+    .row
+      .col-md-12
+        %h2= t :proposals
+      - @motions.each do |motion|
+        .col-md-9
+          - if motion == @discussion.current_motion
+            %h3= t :current_decision
+          %h3= motion.title
+          .proposal-description
+            = motion.description
+          .proposal-details
+            %ul
+              - if motion.voting?
+                %li= t(:closing) + ": " + motion.closing_at.strftime("%d %B, %I.%M %p %Z")
```

Comment:
Hm, this seems like a common thing we're running into: we're duplicating passing parameters because we have to get the right translation key.

Some possible options:
- Switch on the translation key in the view directly
```
t motion.voting? ? :closing : :closed_at, closing_at: motion.closing_at.strftime
```
- Write a helper method for each one
```
t closing_key_for(motion, closing_at: motion.closing_at)

# in helper
def closing_key_for(motion)
  motion.voting? ? :closing : :closed_at
end
```

Writing a helper method which wraps t and allows passing a true/false value and a condition (my favourite)
```
t_ternary(when_true: :closing, when_false: :closed_at, condition: motion.voting?, closed_at: motion.closed_at)

# in helper
def t_ternary(when_true:, when_false:, condition:, *args)
  t condition ? when_true : when_false, args)
end
```